app/helpers/layout_helper.rb

```diff
@@ -124,7 +124,7 @@ def alert_header(text, html_class = nil)
         text ||= _("Error")
     end
     header = icon.to_s
-    header += "<strong>#{text}</strong> " if text.present?
+    header += "<strong>#{text}</strong> ".html_safe if text.present?
```

Comment:
Would be better I think to use `content_tag :strong, text` to prevent possible XSS in case any place may allow user to control the displayed text. 
Also, does the text get correctly translated?